bin/pyadam

```diff
@@ -30,14 +30,14 @@ else
   ASSEMBLY_DIR="$SCRIPT_DIR/adam-assembly/target"
 fi
 
-num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam[0-9A-Za-z\.\_\-]*\.jar$" | grep -v javadoc | grep -v sources | wc -l)"
+num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-assembly_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc | grep -v sources | wc -l)"
```

Comment:
This was fixed a different way in #1651, by delegating to a new `find-adam-assembly.sh` script.  I suppose that doesn't matter if this PR is merged first.